test/GitcoinGovernor.t.sol

```diff
@@ -1321,3 +1351,65 @@ contract FlexVoting is GitcoinGovernorProposalTestHelper, GovernorBravoProposalH
     }
   }
 }
+
+// Exercise the existing Bravo contract deployed on April 7th 2023.
+contract BravoGovernorDeployTestWithExistingBravo is BravoGovernorDeployTest {
+  function _useDeployedGovernorBravo() internal override returns (bool) {
```

Comment:
Nice, I love how cleanly this worked out. Nice job on the refactor.

Reply:
Thanks! :)